ios/safe-area/RNSSafeAreaProviding.h

```diff
@@ -0,0 +1,10 @@
+#import <UIKit/UIKit.h>
+
+#define RNSSafeAreaDidChange @"RNSSafeAreaDidChange"
```

Comment:
I think that notifications-related definitions shouldn't be mixed with the protocols, I'd rather put it in a separate file - maybe one general header file for all definitions related to notifications would be sufficient - the same manner as we have for `RNSEnums`